File: check.py
import re
with open('index.html', 'r', encoding='utf-8') as f:
    text = f.read()
for m in re.finditer(r'<div id="\w+ModalSheet"[^>]+>\s*<div[^>]+padding:(.*?)box-sizing:border-box;', text):
    print(m.group(0))

File: check.js
const fs = require('fs');
const text = fs.readFileSync('index.html', 'utf8');
const rx = /<div id="\w+ModalSheet"[^>]+>\s*<div[^>]+padding:.*?box-sizing:border-box;/g;
let match;
while ((match = rx.exec(text)) !== null) {
    console.log(match[0]);
}